distribution/src/root/config/hazelcast-hardened.yaml

```diff
@@ -0,0 +1,233 @@
+# This is a sample Hazelcast configuration focused on hardened security.
+hazelcast:
+  # Custom cluster name protects against unintended cluster formation between foreign members.
+  cluster-name: ${custom.cluster.name}
+  properties:
+    # Don't listen on all network interfaces, but bind to just one.
+    hazelcast.socket.bind.any: false
```

Comment:
as a user I have to ask: which interface will be picked? at least a link into documentation would help